github/Repository.py

```diff
@@ -1448,6 +1487,162 @@ def __create_pull(self, **kwds):
             self._requester, headers, data, completed=True
         )
 
+    def create_repository_advisory(
```

Comment:
since this is on `Repository`, this could read:
```suggestion
    def create_advisory(
```
Or are there other methods that imply this naming convention?